mesa/scenario.py

```diff
@@ -12,4 +12,12 @@ def from_db_to_dict(variable_name, db):
             table[db.iloc[i, 0]] = [db.iloc[i, 1], db.iloc[i, 2]]
     return table
 
+def topic_definition_list():
```

Comment:
No need for a function here, just create a variable and import it. Unless there are calculations to be done in the future.

Reply:
proton-T prevede la possibilità di modificare i dati demografici in corso d'opera. Io lascerei così e cambierei solo se questa possibilità non si verfica